src/components/Header/logo.js

```diff
@@ -55,17 +55,20 @@ const LogoWrapper = styled.div`
 
 const Logo = styled(({ className, link, img, title }) => {
   const theme = useTheme();
-  let split = title.split(' ');
-  split[0] = '<strong>' + split[0];
-  const last = split.length < 3 ? 0 : split.length - 2;
-  split[last] = split[last] + '</strong>';
-  const title2 = split.join(' ');
+  /*
+    Note: Disable first word bold
+  */
+  // let split = title.split(' ');
+  // split[0] = '<strong>' + split[0];
+  // const last = split.length < 3 ? 0 : split.length - 2;
+  // split[last] = split[last] + '</strong>';
+  // const title2 = split.join(' ');
```

Comment:
Do we still need this code? If not, could you please remove this code comment block?

Reply:
OK.